odl/solvers/iterative/iterative.py

```diff
@@ -404,6 +394,129 @@ def gauss_newton(op, x, rhs, niter=1, zero_seq=exp_zero_seq(2.0),
         if callback is not None:
             callback(x)
 
+
+def kaczmarz(ops, x, rhs, niter=1, omega=1, projection=None,
+             callback=None):
+    """Optimized implementation of Kaczmarz's method.
+
+    Solves the inverse problem given by the set of equations::
+
+        A_n(x) = rhs_n
+
+    This is also known as the Landweber-Kaczmarz's method, since the method
+    coincides with the Landweber method for a single operator.
+
+    Parameters
+    ----------
+    ops : sequence of `Operator`'s
+        Operators in the inverse problem. They must have a
+        `Operator.derivative` property, which returns a new operator which in
+        turn has an `Operator.adjoint` property, i.e.
+        ``op[i].derivative(x).adjoint`` must be well-defined for ``x`` in the
+        operator domain and for all ``i``.
```

Comment:
Comes across a bit convoluted text-wise. I think the part after "i.e." makes perfectly clear what the requirements are and is much more readable; the first part is not needed.